crypto/evp/keymgmt_lib.c

```diff
@@ -138,7 +152,7 @@ void *evp_keymgmt_export_to_provider(EVP_PKEY *pk, EVP_KEYMGMT *keymgmt)
                  */
                 params = paramdefs_to_params(paramdefs);
                 /* Get 'return_size' filled */
-                pk->pkeys[j].keymgmt->exportkey(pk->pkeys[j].provkey, params);
+                exportfn(pk->pkeys[j].provdata, params);
```

Comment:
why is importfn checked for NULL but exportfn isnt?

Reply:
In this particular case, it's all about getting the `return_size` field filled in.  The return value from exportfn is irrelevant here.  However, I've added a check on the next one.